controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -883,6 +883,18 @@ func (r *BareMetalHostReconciler) registerHost(prov provisioner.Provisioner, inf
 		}
 	}
 
+	// Create the hostFirmwareComponents resource with same host name/namespace if it doesn't exist
+	if info.host.Name != "" {
```

Comment:
wondering if we could merge this in the precedent if block as they are based on the same conditions

Reply:
I think it would be ok, let's see what others think about it